source/Preferences.cpp

```diff
@@ -37,7 +37,10 @@ namespace {
 	const string EXPEND_AMMO = "Escorts expend ammo";
 	const string FRUGAL_ESCORTS = "Escorts use ammo frugally";
 
-	const vector<double> ZOOMS = {.25, .35, .50, .70, 1.00, 1.40, 2.00};
+	// This controls the range of zoom scales the player can switch to.
+	// larger values are close-up and small values are farther away
+	const vector<double> ZOOMS = {.1, .2, .3, .4, .5, .6, .7, .8, .9, 1.0,
+		1.1, 1.2, 1.3, 1.4, 1.5, 1.6, 1.7, 1.8, 1.9, 2.0, 3.0, 4.0};
 	int zoomIndex = 4;
```

Comment:
The default zoom is now 0.5.

Reply:
So that zoomIndex is specifying the default zoom level? So shifting it up to 7 would keep the default zoom the same.

Reply:
No, the index of `1.` is now 9.

Reply:
Where does it specify the index? And how does "1." point to 9?

Reply:
Indexing is a property of the programming language.

https://www.cpp.edu/~elab/ECE114/Array.html#:~:text=An%20array%20is%20a%20collection,each%20with%20its%20own%20identifier).

Here's an article that might help.

In programming, "index" is a number that denotes some specific element in a container.
In many programming languages, including C++, container indexing starts at 0, meaning the index of the first element of the container is 0.

The existing vector (`vector` is a specific type of container in C++, it is essentially a standard "array," but its size can be changed dynamically, a standard C++ array does not support this) has these elements at the following indices:

index | 0 | 1 | 2 | 3 | 4 | 5
-- | -- | -- | -- | -- | -- | --
element | .25 | .35 | .5 | 1. | 1.4 | 2.

Your version has:

index | 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 14 | 15 | 16 | 17 | 18 | 19 | 20 | 21
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
element | .1 | .2 | .3 | .4 | .5 | .6 | .7 | .8 | .9 | 1. | 1.1 | 1.2 | 1.3 | 1.4 | 1.5 | 1.6 | 1.7 | 1.8 | 1.9 | 2. | 3. | 4.

Reply:
Ah, yes, I was forgetting that it started at 0, not 1. That makes more sense.